lib/authc/AssertionAuthenticationResult.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var nJwt = require('njwt');
+var Account = require('../resource/Account');
+
+function AssertionAuthenticationResult(dataStore, data) {
```

Comment:
Good question. I actually based it off of JwtAuthenticationResult, and it felt like AuthenticationResult maybe contained more than was needed. @robertjd What do you think?